packages/tokens/package.json

```diff
@@ -12,9 +12,14 @@
     "*.css",
     "*.scss"
   ],
-  "main": "dist/cjs/index.js",
-  "module": "dist/index.js",
-  "types": "dist/src/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/src/index.d.ts",
+      "import": "./dist/index.js",
+      "require": "./dist/cjs/index.js"
+    },
+    "./*": "./*"
```

Comment:
Trenger denne for at det fortsatt skal være mulig å gjøre imports av f.eks. css-filer, sånn som `import '@norges-domstoler/dds-design-tokens/dist/css/designTokens.css'`

Reply:
👍